docker/iceberg-rest-adapter-image/README.md

```diff
@@ -0,0 +1,86 @@
+                      ├── 6: c_first_shipto_date_sk: optional int                                                                                                               
+                      ├── 7: c_first_sales_date_sk: optional int                                                                                                                
+                      ├── 8: c_salutation: optional string                                                                                                                      
+                      ├── 9: c_first_name: optional string                                                                                                                      
+                      ├── 10: c_last_name: optional string                                                                                                                      
+                      ├── 11: c_preferred_cust_flag: optional string                                                                                                            
+                      ├── 12: c_birth_day: optional int                                                                                                                         
+                      ├── 13: c_birth_month: optional int                                                                                                                       
+                      ├── 14: c_birth_year: optional int                                                                                                                        
+                      ├── 15: c_birth_country: optional string                                                                                                                  
+                      ├── 16: c_login: optional string                                                                                                                          
+                      ├── 17: c_email_address: optional string                                                                                                                  
+                      └── 18: c_last_review_date: optional string                                                                                                               
+Snapshots             Snapshots                                                                                                                                                 
+                      └── Snapshot 0: s3://iceberg-test-data/tpc/tpc-ds/3.2.0/1000/iceberg/customer/metadata/snap-643656366840285027-1-5ce13497-7330-4d02-8206-7e313e43209c.avro
+Properties            write.object-storage.enabled  true                                                                                                                        
+                      write.object-storage.path     s3://iceberg-test-data/tpc/tpc-ds/3.2.0/1000/iceberg/customer/data
+```
+
```

Comment:
For a separate PR, can document how we generate the `LICENSE` and `NOTICE`?

cc @ajantha-bhat @bryanck

Reply:
Sure, I was planning to add the grade task to regenerate it soon.

Reply:
Yes, what we need is a plugin or gradle task for each runtime jar module, that automatically updates the notice and license on every build. Manually doing it is not efficient. 

Because for example, after AWS dependency version bump by dependabot, it may bring new dependencies and our notice and license may never get updated for it.

Reply:
created an issue tracker for this https://github.com/apache/iceberg/issues/11559

Reply:
I will help on the `LICENSE` and `NOTICE` as it's bin distribution here (so slightly different from source distribution).

Reply:
Ack